src/main/java/racingcar/domain/Cars.java

```diff
@@ -0,0 +1,48 @@
+package racingcar.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Cars {
+    private final List<Car> cars;
+
+    public Cars(int numberOfCars) {
+        List<Car> cars = new ArrayList<>();
+        for (int i = 0; i < numberOfCars; i++) {
+            cars.add(new Car());
+        }
+        this.cars = cars;
+    }
+
+    public Cars(List<Car> cars) {
+        this.cars = cars;
+    }
+
+    public Cars moveAllCar() {
+        List<Car> newCars = new ArrayList<>();
+        for (int carNumber = 0; carNumber < cars.size(); carNumber++) {
+            Car movedCar = cars.get(carNumber)
+                    .move(new MoveStrategy(
+                            new RandomCondition().getRandomCondition()));
```

Comment:
```suggestion
                    .move(moveStrategy);
```
MoveStrategy를 생성자를 통해 주입하는 것은 어떨까요
그리고 Strategy라는 interface를 도출하여 테스트에서는 true 조건을 stub한다면, 테스트코드를 작성하기 더 유연해질거라 생각합니다.